src/Halogen/Component.purs

```diff
@@ -58,6 +60,8 @@ import Halogen.Query.EventSource (EventSource(..), ParentEventSource(), runEvent
 import Halogen.Query.HalogenF (HalogenF(), HalogenFP(..), hoistHalogenF, transformHF)
 import Halogen.Query.StateF (StateF(..), mapState)
 
+import Unsafe.Coerce (unsafeCoerce)
```

Comment:
Wait, what now? :laughing:  (I assume this is leftover from hole driven development)